apps/charterafrica/src/lib/tools/fetchGithub.js

```diff
@@ -0,0 +1,77 @@
+import fetchJson, { FetchError } from "@/charterafrica/utils/fetchJson";
```

Comment:
May be this should be lib/github or lib/tools/github. Can't be `fetGithub`. We are not fetching GH, we're fetching _from_ GH/